integration_test/sql/migration.exs

```diff
@@ -349,6 +358,12 @@ defmodule Ecto.Integration.MigrationTest do
     assert :ok == down(PoolRepo, 20050906120000, InferredDropIndexMigration, log: false)
   end
 
+  @tag :very_long_index
+  test "create very long index name" do
```

Comment:
Unfortunately I haven't been able to make Postgrex return a hint yet 🤔. Anyone has a hint?

Reply:
Here is a migration that emits a warning:

```
    def change do
      create_if_not_exists table(:duplicate_table)
      create_if_not_exists table(:duplicate_table)
    end
```

but this does not work right now. So I will push a fix for it and then you can try again on master.

Also, this is very likely a PG specific warning, so we will have to move this test to inside `integration_test/pg/migrations_test.exs`. See the existing `constraints_test.exs` for an example.

Reply:
Ok, [I pushed to master](29adbb95). The migration above should work and emit warnings.

Reply:
Thanks! I'll fix this.